kairon/shared/channels/whatsapp/bsp/dialog360.py

```
@@ -281,4 +283,88 @@
+
+        base_url = Utility.system_metadata["channels"]["whatsapp"]["business_providers"]["360dialog"]["waba_base_url"]
+        auth_header = Utility.system_metadata["channels"]["whatsapp"]["business_providers"]["360dialog"]["auth_header"]
+
+        headers = {auth_header: access_token}
+        payload = {"messaging_product": "whatsapp"}
+        content_dir = os.path.join("media_upload_records", bot)
+        os.makedirs(content_dir, exist_ok=True)
+        file_path = os.path.join(content_dir, filename)
 
+        files = {
+            "file": (filename, open(file_path, "rb"), f"{extension}")
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)